h2o-bindings/bin/custom/python/gen_gbm.py

```diff
@@ -3,6 +3,19 @@ def update_param(name, param):
         param['values'].remove('ordinal')
         return param
     return None  # param untouched
+ 
+def class_extensions():
```

Comment:
do we want the same for other tree algos as well? drf, xgboost, isofor?

Reply:
not specified on the ticket but clarifying in #h2o-3-r-client

Reply:
added for all tree algos